addons/forum-live-preview/userscript.js

```diff
@@ -1,6 +1,7 @@
 export default async function ({ addon, console }) {
   const textarea = await addon.tab.waitForElement(".markItUpEditor");
   const previewButton = await addon.tab.waitForElement(".markItUpButton.preview");
+  const markItUpButtons = document.querySelectorAll(".markItUpButton:not(.markItUpDropMenu)");
```

Comment:
No guarantee the element exists yet

Reply:
Actually, nevermind, the preview button is the last one and the line above waits until it exists
Not sure how this would play with other addons that add their own buttons to the toolbar though

Reply:
> Not sure how this would play with other addons that add their own buttons to the toolbar though

True, an event listener on body that checks the `classList` would probably be better here

Reply:
Never mind, Scratch seems to call `e.stopPropagation()` or something to that effect when a button is clicked